examples/simple/posix/UB-PTHREAD2-bad.output

```diff
@@ -0,0 +1,12 @@
+Error: UB-PTHREAD2
+Description: Destroying a locked mutex.
+Locked here:
+  at pthread_mutex_lock(UB-PTHREAD2-bad.c:15)
+  at locker_thread(UB-PTHREAD2-bad.c:15)
+  at <file-scope>(<unknown>)
+
```

Comment:
@msaxena2 go ahead and remove this line and I will fix the code in the upcoming pull request I am working on that deals with examples.